payroll_contract_advantages/README.rst

```diff
@@ -0,0 +1,90 @@
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpayroll-lightgray.png?logo=github
+    :target: https://github.com/OCA/payroll/tree/14.0/payroll_contract_advantages
+    :alt: OCA/payroll
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/payroll-14-0/payroll-14-0-payroll_contract_advantages
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/281/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds support for advantages templates and advantages to be set in contract form.
+The advantages can be set in the contract form as a list of advantages templates.
+Then it can be used in the calculation of the salary rules.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+- Set the advantages templates in the payroll module with lower and upper bounds and default value.
+- Go to the employee contract and add the advantages that you want for this contract, default value will be populated but you can change it.
+- Then in the salary rules, access this value using current_contract.advantages.[ADVANTAGE_CODE] (without brackets)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/payroll/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/payroll/issues/new?body=module:%20payroll_contract_advantages%0Aversion:%2014.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Nimarosa
+
+Contributors
+~~~~~~~~~~~~
+
+* Nimarosa (Nicolas Rodriguez) <nicolasrsande@gmail.com>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+.. |maintainer-nimarosa| image:: https://github.com/nimarosa.png?size=40px
+    :target: https://github.com/nimarosa
+    :alt: nimarosa
+
+Current `maintainer <https://odoo-community.org/page/maintainer-role>`__:
+
+|maintainer-nimarosa| 
+
+This module is part of the `OCA/payroll <https://github.com/OCA/payroll/tree/14.0/payroll_contract_advantages>`_ project on GitHub.
```

Comment:
Readme files are still on version 14.0

Reply:
I thought that readmes will be automatically updated on merge. Will update them.